src/player-gui.js

```diff
@@ -12,7 +12,8 @@ const mapStateToProps = state => ({
     shell: state.shell,
     engine: {
       adBreak: state.engine.adBreak,
-      isLive: state.engine.isLive
+      isLive: state.engine.isLive,
+      error: state.engine.error
```

Comment:
change name to `hasError`

Reply:
I think a state named 'error' is pretty clear.